drivers/include/bq2429x.h

```diff
@@ -330,6 +330,9 @@ int bq2429x_get_fault(const bq2429x_t *dev, bq2429x_fault_t *fault);
 /**
  * @brief   Enable OTG.
  *
+ * This allows powering USB devices from the same port, i.e. to provide power
+ * or charge other devices.
```

Comment:
So this switches between charging (OTG off) and powering devices via USB (OTG on)?

Reply:
Yes, exactly, charging will be disabled, (even if the charge enable pin is on and I2C charge enable config is on too). Let me add this to the docs too.